litex/soc/software/bios/boot.c

```diff
@@ -307,6 +305,118 @@ const char *filename, char *buffer)
 	return size;
 }
 
+#ifdef ETH_UDP_DYNAMIC_IP
+
+static uint8_t set_ip(const char * ip_address, unsigned int * ip_to_change)
```

Comment:
I'm wondering about the name of this function; it doesn't `set` anything, rather just parses string into an array.

Reply:
I have renamed the function to `parse_ip` since that better explain what it does